### ISSUE TITLE
Python prefixed formats

### ISSUE THREAD

**Issue by Zarad1993** [Zarad1993/dyc#50]
We have a default config file for defining formats. It would be nice to create known formats and reference them in our `dyc.yaml` file.

### Usage
`dyc.yaml`
_Just an idea_

```
-
  extension: py
  style: sphinx/google/numpy
```
Then it would follow the known style format of sphinx or google or numpy

We can have these files in defaults for references
`configs/{{style}}.yaml`

That way it would be easier for a project to set docstrings standards based on a style

**Comment by Shankhanil** [comment]
I am interested in this. May I take a look? @Zarad1993 . Just help me out in case I need assistance

**Comment by Zarad1993** [comment]
@Shankhanil - Yes, please. Feel free to star tackling it 🙂

**Comment by Shankhanil** [comment]
Can you pls go ahead and detail the issue. What exactly do you require, and which portion of codebase do i need to go through. 
Btw, I am using dyc in my new python project! @Zarad1993

**Comment by Zarad1993** [comment]
That's great. Sure I'll add some more details soon. The high-level idea is to add some fixed templates, so instead of customizing it. You can choose pre-existing ones.
This is an example of a sphinx ones https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html

**Comment by Shankhanil** [comment]
Cool, so I am going to add sphinx docstring format initially, and if that's successful, I will introduce NumPy and Google style. @Zarad1993

**Comment by Zarad1993** [comment]
Yes, then instead of having to define these formats manually in your project. You can just extend it to use them across the board.

**Comment by Shankhanil** [comment]
I really apologize for the impossible delay. I took some time to understand the project, and then caught busy with other stuffs. Please review the PR. @Zarad1993

**Comment by Shankhanil** [comment]
the default format given for DYC is only documenting the function and the parameters. But it doesn't specify a format for a `class` and `errors` that the function is raising.  Is this work under progress?

**Comment by ajjayymahato** [comment]
Is this issue still open? I would like to work on this issue. Can you please assign this to me?

**Comment by yuval728** [comment]
Hey, I'm new to open source and would like to contribute.
